### ISSUE TITLE
King County COVID19 - Bug: Language attributes are missing for the languages in the middle of the page

### ISSUE THREAD

**Issue by lamarjordanux** [a11ySea/testing#6]
## Issue Summary

Screen readers may not announce the language properly without lang attributes.

## Steps to reproduce

1. Go to one of the language buttons in the middle of the page
2. Inspect the button code 

### Expected behavior

Screen reader pronounces the language in native tongue.

### Actual behavior

- Screen reader pronounces language in English 

### Device Information:

 - OS: NVDA
 - Browser Chrome

### Screen Reader or AT Information

 - [e.g. JAWS 2020.2003.13]
 - [e.g. NVDA 2019.3.1]

### Current Code
```
(code snippet)
```

### Suggested Code
```
(code snippet)
```

# Specifications

- **WCAG Principal:**
- **WCAG SC:**
- **Severity:**

## Additional References

- Link to any articles or specs that give further guidance

**Comment by dpersing** [comment]
<img width="873" alt="Screenshot of the language controls" src="https://user-images.githubusercontent.com/1462085/80555237-8ea3a880-8984-11ea-9bcc-b58ce9c5f51a.png">

**Comment by alboss** [comment]
Current code:
`<a class="btn btn-primary btn-block" href="/depts/health/covid-19/languages/spanish.aspx" role="button" data-toggle="tooltip" data-placement="left" title="" data-original-title="COVID-19 Recomendaciones de Salud Pública">Spanish <span class="fas fa-long-arrow-alt-right" aria-hidden="true"></span> Español</a>`

Suggested code:
`<a class="btn btn-primary btn-block" href="/depts/health/covid-19/languages/spanish.aspx" role="button" data-toggle="tooltip" data-placement="left" title="" data-original-title="COVID-19 Recomendaciones de Salud Pública">Spanish <span class="fas fa-long-arrow-alt-right" aria-hidden="true"></span> <span lang="es">Español</span></a>`

What changed?
`Español` is now `<span lang="es">Español</span>` so a text reader will read it as a Spanish word instead of sounding it out as an English word and mispronouncing it.

Impact: Essential for text readers.

Difficulty of change: tedious but not difficult.
